modules/distr-tracing-product-overview.adoc

```diff
@@ -21,8 +21,10 @@ With {DTShortName} you can perform the following functions:
 
 * Perform root cause analysis
 
-{DTProductName} consists of two main components:
+{DTProductName} consists of three components:
 
-* *{JaegerName}* - This component is based on the open source link:https://www.jaegertracing.io/[Jaeger project].
+* *{JaegerName}*, which is based on the open source link:https://www.jaegertracing.io/[Jaeger project].
 
-* *{OTELNAME}* - This component is based on the open source link:https://opentelemetry.io/[OpenTelemetry project].
+* *{TempoName}*, which is based on the open source link:https://github.com/grafana/tempo[Grafana Tempo project].
```

Comment:
```suggestion
* *{TempoName}*, which is based on the open source link:https://grafana.com/oss/tempo/[Grafana Tempo project].
```

Changed link target to grafana.com instead of github.com